tests/test_formula_parser.py

```diff
@@ -169,5 +153,16 @@ def test_average(self):
         result = func({"a1": torch.tensor([2, 4, 8]), "a2": torch.tensor([3, 4, 5])})
         assert (torch.abs(result - torch.tensor([2.66666666, 4, 6])) < 0.000001).all()
 
+    def test_min_max_array(self):
+        p = Parser(debug=True)
+        func = p.parse("MIN(a1, a2)")["result"]
+        result = func({"a1": torch.tensor([5, 5, 5]), "a2": torch.tensor([1, 10, 5])})
+        assert (torch.abs(result - torch.tensor([1, 5, 5])) < 0.00001).all()
+
+        p = Parser(debug=True)
+        func = p.parse("MAX(a1, a2)")["result"]
+        result = func({"a1": torch.tensor([5, 5, 5]), "a2": torch.tensor([1, 10, 5])})
+        assert (torch.abs(result - torch.tensor([5, 10, 5])) < 0.00001).all()
```

Comment:
it's probably worth writing some assertion helpers to reduce testing boilerplate next time you work on these tests